drivers/watchdog/Kconfig.ti_tps382x

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Prevas A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config WDT_TI_TPS382X
+	bool "External TI TPS382x Watchdog (WDT) Driver"
+	default y
+	depends on DT_HAS_TI_TPS382X_ENABLED
```

Comment:
It seems that you need to add `select GPIO` since it's an external watchdog controlled by GPIO.

Reply:
Fixed. Using `depends on` since it is a lot more common:
```
$ for w in select imply "depends on"; do echo $w; grep -Pr "$w GPIO[^_]*$" --include=Kconfig* | wc --lines; done
select
5
imply
3
depends on
86
```

Reply:
Right, `select X` is mostly used in the drivers area.